common/logger/src/security.rs

```diff
@@ -26,16 +26,10 @@ pub fn security_log(name: &'static str) -> StructuredLogEntry {
 
 /// Security events that are possible
 pub mod security_events {
-    // JSON-RPC server
-    // ---------------
-
-    /// Admission Control received a transaction with an invalid signature
-    pub const INVALID_TRANSACTION_AC: &str = "InvalidTransactionAC";
```

Comment:
I don't think we should log this, as we're probably going to receive a ton of invalid transaction via the json-rpc interface